docs: added section for community meetings in the Readme

rujutashinde opened the pull request:
added dir for meeting recordings
added section on the main readme page

Firewall reviewed (approved):
👍 

You need to update to semantic commit msg though

rujutashinde commented:
> 👍
> 
> You need to update to semantic commit msg though

updated that. and added latest meeting as a tag to be able to refer to the latest meeting
